### ISSUE TITLE
Cannot install on MacOS 12.3.1

### ISSUE THREAD

**Issue by jjriffle** [pawelsalawa/sqlitestudio#4450]
When clicking the download to install I receive this message on both my Apple Chip iMac and Intel Chip MacBook Pro. 

“SQLiteStudio” cannot be opened because the developer cannot be verified.
macOS cannot verify that this app is free from malware.
This item is on the disk image “sqlitestudio-3.3.3.dmg”. Safari downloaded this disk image today at 10:57 AM from objects.githubusercontent.com.

### Steps to reproduce
Download Mac version
Click download to install

### Operating system
macOS Monterey 12.3.1

### SQLiteStudio version
3.3.3

**Comment by pawelsalawa** [comment]
Right-click on the app, choose "Open". Then in the same warning dialog you will have an option (a button in bottom corner) to "Open" the app anyway.
